svcs/guacamole/intercept.go

```diff
@@ -6,14 +6,228 @@ package guacamole
 
 import (
 	"errors"
-	"fmt"
 	"net/http"
 	"sync"
 
 	"github.com/aau-network-security/haaukins/store"
 	"github.com/rs/zerolog/log"
 )
+const waitingHTMLTemplate = `
```

Comment:
A fancy waiting UI added as constant, I think, it could be useful to define someplace where all html & css stuff resides. Seems dirty here however it is fancy on browser :)